src/seedu/addressbook/common/PersonComparator.java

```diff
@@ -0,0 +1,19 @@
+package seedu.addressbook.common;
+
+import seedu.addressbook.data.person.Person;
+
+import java.util.Comparator;
+
+/**
+ * Comparator class exclusively for comparing persons
+ * in the address book by their names
+ */
+public class PersonComparator implements Comparator<Person>{
+    public PersonComparator(){
+    }
+
+    @Override
+    public int compare(Person o1, Person o2) {
```

Comment:
can change the naming from 'o1' and 'o2' to 'firstPerson' and 'secondPerson' for better readability